tests/test_ingesters.py

```diff
@@ -443,6 +460,27 @@ def test_ingest_dataset_twice_different_urls(self):
         self.assertTrue(logger_cm.records[0].msg.endswith('already exists in the database.'))
         self.assertEqual(Dataset.objects.count(), initial_datasets_count + 1)
 
+    def test_function_named_prepare_url(self):
+        """ test the functionality of 'prepare_url' for a ddxingester """
+        input_url = 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_NPP/NAVO/v1/2014/005/20140105235906-NAVO-L2P_GHRSST-SST1m-VIIRS_NPP-v02.0-fv01.0.nc'
+        output_url = 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_NPP/NAVO/v1/2014/005/20140105235906-NAVO-L2P_GHRSST-SST1m-VIIRS_NPP-v02.0-fv01.0.nc.ddx'
+        ingester = ingesters.DDXIngester()
+        self.assertEqual(output_url, ingester.prepare_url(input_url))
+
+    def test_function_named_prepare_url2(self):
+        """ no change when a ddx file has been given to the function """
+        input_url2 = 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_NPP/NAVO/v1/2014/005/20140105235906-NAVO-L2P_GHRSST-SST1m-VIIRS_NPP-v02.0-fv01.0.nc.ddx'
+        output_url2 = 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_NPP/NAVO/v1/2014/005/20140105235906-NAVO-L2P_GHRSST-SST1m-VIIRS_NPP-v02.0-fv01.0.nc.ddx'
```

Comment:
it's a different method, you don't need to change the variable names

Reply:
Done in the next commit.